src/extension/helper_process.c

```diff
@@ -43,6 +44,8 @@ typedef struct _dd_helper_mgr {
     uint16_t failed_count;
     bool connected_this_req;
     bool launched_this_req;
+    char *nonnull socket_path;
```

Comment:
This structure could be better compacted if the pid was put after the second bool.